src/dft/backends/cufft/commit.cpp

```diff
@@ -323,6 +351,76 @@ class cufft_commit final : public dft::detail::commit_impl<prec, dom> {
         return offsets;
     }
 
+    virtual void set_workspace(scalar_type* usmWorkspace) override {
```

Comment:
I think we should stick to snake_case for function arguments, same for `bufferWorkspace`.

Reply:
That would be consistent with the contribution guidelines for oneMKL. I've updated your suggestions here, and a few other places I missed in [1da13ee](https://github.com/oneapi-src/oneMKL/pull/411/commits/1da13eef0fbb663fcadc1abbcc15ad930f977589).

Reply:
There are a few more occurrences of `bufferWorkspace` and `usmWorkspace` in other files.

Reply:
Hopefully all the stragglers caught in [fd528ea](https://github.com/oneapi-src/oneMKL/pull/411/commits/fd528eae5034fec9ca50f349c74f8c7b3925f443).